index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home</title>
+  <link rel="stylesheet" href="pages/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;700&display=swap" rel="stylesheet">
+
+  
+</head>
+<body>
+
+<header>
+    <nav>
+      
+        <!-- Main Navigation inside <header> -->
+        <ul class="menu">
+            <!-- <li id="logo"><a href="index.html">X</a></li> -->
+            <li><span>✨</span></li>
+            <li><a href="pages/about.html">About</a></li>
+            <li><span>✨</span></li>
+            <li><a href="pages/portfolio.html">Portfolio</a></li>
+            <li><span>✨</span></li>
+        </ul>
+    </nav>
+</header>
+
+<main class="lorem">
+    <div class="heading"><h1>Hola, I'm 
+      <a href="pages/about.html" class="about-link">Xiomara</a></h1>
+    </div>
+      <p>
+        I'm a software dev based in NYC, Lorem ipsum dolor sit amet, 
+        consectetur adipisicing elit. Quidem, eligendi reprehenderit. 
+        Facilis obcaecati laborum quo cum odit, sint provident ipsum 
+        ratione mollitia officiis minima placeat dolorem perferendis.
+        <br>
+        <br>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+        Quis doloremque dolores animi, corporis iste pariatur at 
+        natus aliquam quae temporibus a expedita. Dolores possimus 
+        cupiditate totam labore perferendis pariatur nemo.
+      </p>
+</main>
+
+<footer>
+  <h2>Let's link up</h2>
+  <nav>
+    <!-- Main Navigation inside <footer> -->
+    <ul><li><a href="https://www.gmail.com/"><img src="images/google.png"></li>
+      <li class="github"><a href="https://github.com/xiomaraR"><img src="images/Octocat.png"></a></li>
```

Comment:
Looks like we have a few errors here when we pass this HTML through a validator.

I think we are missing a closing `a` tag:

```suggestion
    <ul>
      <li><a href="https://www.gmail.com/"><img src="images/google.png" alt="something here"></a></li>
      <li class="github"><a href="https://github.com/xiomaraR"><img src="images/Octocat.png" alt="something here"></a></li>
```

and don't forget your `alt` attributes inside the `img` tag!